test/built-ins/Temporal/Instant/prototype/toZonedDateTimeISO/to-zoned-date-time-iso.js

```diff
@@ -0,0 +1,28 @@
+// Copyright (C) 2022 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-temporal.instant.prototype.tozoneddatetimeiso
+description: Test time zone parameters.
+includes: [temporalHelpers.js]
+features: [Temporal]
+---*/
+
+const inst = Temporal.Instant.from("1976-11-18T14:23:30.123456789Z");
+
+// throws without parameter
+assert.throws(TypeError, () => inst.toZonedDateTimeISO());
+
+// time zone parameter UTC
+const zdt = inst.toZonedDateTimeISO("UTC");
+assert.sameValue(inst.epochNanoseconds, zdt.epochNanoseconds);
+TemporalHelpers.assertZonedDateTimesEqual(
+    zdt,
+    Temporal.ZonedDateTime.from("1976-11-18T14:23:30.123456789+00:00[UTC]"));
```

Comment:
What I'd suggest here (and below) is, since you've already determined that the epoch nanoseconds are the same, to just check the time zone:
```suggestion
assert.sameValue(zdt.timeZoneId, "UTC");
```